sql/core/src/main/scala/org/apache/spark/sql/execution/python/UserDefinedPythonFunction.scala

```diff
@@ -190,6 +190,8 @@ class UserDefinedPythonTableFunctionAnalyzeRunner(
 
   override val workerModule = "pyspark.sql.worker.analyze_udtf"
 
+  private lazy val parser = new CatalystSqlParser()
```

Comment:
We should use a parser in `SparkSession`.

Reply:
This is done.